feature/image/src/main/kotlin/net/twisterrob/colorfilters/android/image/ImageFragment.kt

```diff
@@ -62,16 +64,23 @@ class ImageFragment : Fragment() {
 		listener = context as Listener
 	}
 
-	override fun onCreate(savedInstanceState: Bundle?) {
-		super.onCreate(savedInstanceState)
-		setHasOptionsMenu(true)
-	}
-
 	override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View =
```

Comment:
looks strange ![image](https://user-images.githubusercontent.com/2906988/178165042-551c108a-1b77-460e-bd63-ef412240e786.png)

Reply:
it was like this since at least latest stable 3.0